content/altinn-studio/app-creation/logic/_index.md

```diff
@@ -287,9 +287,37 @@ Et eksempel på en egendefinert validering der headerverdien hentes ut er vist n
           validationResults.AddModelError(value[0], "Dette er ikke en gyldig kommune.");
       }
     }
+
+    await Task.CompletedTask;
  }
 ```
 
+Det er gjort flere ting for å få denne kodesnutten til å kjøre
+
+1. I _ValidationHandler.cs_ inkluderes `using Microsoft.Extensions.Privites;` øverst i filen for å kunne ta i bruk `StringValues`. 
+2. I _App.cs_ inkluderes `using Microsoft.AspNetCore.Http;` øverst i filen for å kunne ta i bruk `IHttpContextAccessor`.
+3. I _App.cs_ dependency injectes `IHttpContextAccessor` i konstruktøren og sendes med videre til ValidationHandler.
+
+```cs
```

Comment:
@acn-sbuad Add highlight of the two lines in the code, makes it much more readable